rust/toolchain.bzl

```diff
@@ -247,6 +247,8 @@ rust_toolchain = rule(
         "staticlib_ext": attr.string(mandatory = True),
         "dylib_ext": attr.string(mandatory = True),
         "os": attr.string(mandatory = True),
+        "exec_triple": attr.string(mandatory = True),
+        "target_triple": attr.string(mandatory = True),
```

Comment:
Making these mandatory is a breaking change for anyone with a custom toolchain, may want to hold off

Reply:
I think these will need to be mandatory very soon (as soon as when target_triple is passed to rustc), but I can make it optional now. Done.